docs/maui/behaviors/touch-behavior.md

```diff
@@ -267,3 +267,94 @@ You can find an example of this behavior in action in the [.NET MAUI Community T
 ## API
 
 You can find the source code for `TouchBehavior` over on the [.NET MAUI Community Toolkit GitHub repository](https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui/Behaviors/TouchBehavior.shared.cs).
+
+
+## Migrating From Xamarin Community Toolkit
+
+In the [Xamarin Community Toolkit](https://github.com/xamarin/XamarinCommunityToolkit), there was the [`TouchEffect`](https://learn.microsoft.com/en-us/dotnet/api/xamarin.communitytoolkit.effects.toucheffect?view=xamarin-community-toolkit-sdk) if you are upgrading an app from Forms to Maui there are a number of changes that you need to be aware of.
+
+The XCT implementation was an attached effect and was applied to an element in the following way:
+
+```xaml
+<StackLayout Orientation="Horizontal"
+            HorizontalOptions="CenterAndExpand"
+            xct:TouchEffect.AnimationDuration="250"
+            xct:TouchEffect.AnimationEasing="{x:Static Easing.CubicInOut}"
+            xct:TouchEffect.PressedScale="0.8"
+            xct:TouchEffect.PressedOpacity="0.6"
+            xct:TouchEffect.Command="{Binding Command}">
+    <BoxView Color="Gold" />
+    <Label Text="The entire layout receives touches" />
+    <BoxView Color="Gold"/>
+</StackLayout>
+```
+
+You should be aware of the 2 following major changes to this effect:
+1. `TouchBehavior` is now a `PlatformBehavior`, not an `AttachedEffect`
+2. The `BindingContext is not automatically set
```

Comment:
```suggestion
2. The `BindingContext` is not automatically set
```

Reply:
Can we maybe also write a little bit more about what that means exactly? More experienced developers will know, newcomers will maybe have trouble understanding what the symptoms are they might be seeing and what problem this solves.

Doesn't have to be super extensive, just something like: that means data-binding will not automatically work and pick up values in your view models.

I see you're expanding on this a little bit later? Always good to add a reference.

Reply:
I've turned this article into 2 sections and put links to each in this bullet list so it is clear about each point